modules/likes.php

```diff
@@ -621,17 +621,20 @@ function jetpack_post_likes_update_value( $enable_post_likes, $post_object ) {
  * @link https://developer.wordpress.org/rest-api/extending-the-rest-api/modifying-responses/
  */
 function jetpack_post_likes_register_rest_field() {
-	register_rest_field(
-		'post', 'jetpack_likes_enabled',
-		array(
-			'get_callback' => 'jetpack_post_likes_get_value',
-			'update_callback' => 'jetpack_post_likes_update_value',
-			'schema' => array(
-				'description' => __( 'Are Likes enabled?' ),
-				'type'        => 'boolean'
-			),
-		)
-	);
+	$post_types = get_post_types( array( 'public' => true ) );
+	foreach( $post_types as $post_type ) {
+		register_rest_field(
+			$post_type, 'jetpack_likes_enabled',
+			array(
+				'get_callback' => 'jetpack_post_likes_get_value',
+				'update_callback' => 'jetpack_post_likes_update_value',
+				'schema' => array(
+					'description' => __( 'Are Likes enabled?' ),
```

Comment:
You'll want to add the `jetpack` textdomain here.

```suggestion
					'description' => __( 'Are Likes enabled?', 'jetpack' ),
```